packages/admin-ui-docs/content/tooltip/code.mdx

```diff
@@ -0,0 +1,48 @@
+import { Section } from '@site/src/components/Section'
+import { OrderedList, OrderedListItem } from '@site/src/components/OrderedList'
+
+<Section>
+
+## Usage
+
+```jsx isStatic
+import { Tooltip } from '@vtex/admin-ui'
+
+function Example() {
+  return <Tooltip text="Tooltip text" />
+}
+```
+
+</Section>
+
+<Section>
+
+## Tooltip Props
+
+| Name      | Type                                                                                | Description                                           | Required | Default |
+| --------- | ----------------------------------------------------------------------------------- | ----------------------------------------------------- | -------- | ------- |
+| text      | `string`                                                                            | Text shown inside the tooltip                         | ✅       | -       |
+| children  | `ReactNode`                                                                         | The element that triggers the tooltip                 | 🚫       | -       |
+| placement | `auto-start, auto, auto-end, top-start, top, top-end, right-start, bottom, or left` | The placement of the tooltip relative to its children | 🚫       | `top`   |
+| visible   | `boolean`                                                                           | Whether the tooltip is visible or not                 | 🚫       | `false` |
+| csx       | `StyleProps`                                                                        | Custom styles                                         | 🚫       | `{}`    |
+| bleedX    | `boolean`                                                                           | Horizontal bleed                                      | 🚫       | `false` |
+| bleedY    | `boolean`                                                                           | Vertical bleed                                        | 🚫       | `false` |
+
+</Section>
+
+<Section direction="row">
+
+## Accessibility
+
+<OrderedList>
+  <OrderedListItem>`Tooltip` has `role="tooltip"`.</OrderedListItem>
```

Comment:
These ` (plicks or crases) are buggy on the live example. Don't know why

Reply:
I think it is the way OrderedListItem is implemented. We will need to update it to be able to accept the markdown language., probably similar to what we did to `<Section/>` component

Reply:
So, I will leave it here